### DISCUSSION TITLE
Compilation fails: ModuleNotFoundError: No module named 'encodings'

### DISCUSSION THREAD

**Discussion by Macrosii** [earlephilhower/arduino-pico#1284] [Q&A] [answered]
Windows 10
Raspberry Pi Pico Arduino core Version 3.0
IDE 1.8.19
Boards any

Hi on trying to compile any code I get the following error;
"Fatal Python error: initfsencoding: unable to load the file system codec
ModuleNotFoundError: No module named 'encodings'
Current thread 0x00004324 (most recent call first):
exit status -1073740791
Error compiling for board Raspberry Pi Pico."

I can compile ok with the same installation on Teensy, Feather, ESP32, AVR etc.
Thanks for deleloping this core and for any help in advance.

**Comment by earlephilhower** [comment, marked as answer]
This is an IDE 1.x issue as far as we can tell.  The initial install crashes while extracting Python causing the issue.

You need to delete the board from the Boards Manager and reinstall.  https://github.com/arduino/Arduino/issues/11842

**Reply by Audi-V** [reply]
worked, thank you!!

**Comment by Macrosii** [comment]
Wow, that was a quick reply.
Yes that did it.
Many thanks and regards
